src/ElvAccount.js

```diff
@@ -19,18 +19,32 @@ class ElvAccount {
     this.debug = debugLogging;
   }
 
-  async Init({ privateKey }) {
+  async Init({ privateKey, tenantCheck = true }) {
     this.client = await ElvClient.FromConfigurationUrl({
       configUrl: this.configUrl,
     });
     this.wallet = this.client.GenerateWallet();
     this.signer = this.wallet.AddAccount({
       privateKey, 
     });
-    this.client.SetSigner({ signer:this.signer });
+    this.client.SetSigner({ signer:this.signer, tenantCheck });
     this.client.ToggleLogging(this.debug);
   }
 
+  async InitWithId({ privateKey, id }) {
```

Comment:
I think it is best to remove the check from SetSigner() - because that code could be used for other things, not necessarily fabric tenant operations.

In that case this function can just call Init() and then this.SetAccountTenantContractId (to avoid duplicating the code)